src/ConnectedMode/SlCoreConnectionAdapter.cs

```diff
@@ -136,6 +137,13 @@ public Task<AdapterResponseWithData<ServerProject>> GetServerProjectByKeyAsync(I
         });
     }
 
+    public async Task<AdapterResponseWithData<List<ServerProject>>> GetAllProjectsAsync(ConnectionInfo connectionInfo, ICredentials credentials)
+    {
+        var credentialsDto = MapCredentials(credentials);
+        var validateConnectionParams = GetAllProjectsParams(connectionInfo, credentialsDto);
+        return await GetAllProjectsAsync(validateConnectionParams);
```

Comment:
Create a ticket to find a solution/support more than 10000 projects

Reply:
https://sonarsource.atlassian.net/browse/SLVS-1456